packages/selinux-policy/object.cil

```diff
@@ -138,6 +138,11 @@
 (roletype object_r secret_t)
 (context secret (system_u object_r secret_t s0))
 
+; Files for saved system secrets in config files in /etc.
```

Comment:
nit on the commit messages:

Both messages should have an `selinux-policy: ` prefix.

For the second commit, bootstrap commands have a different permissions model. They are executed by the system in response to API commands. So they won't be affected by this.

Reply:
updated in latest force push